src/message.ts

```diff
@@ -613,6 +633,81 @@ export class Message implements Sayable {
   // }
 
   /**
+   *
+   * @param sendTo Room or Contact
+   * @param option Whether to display the message source, and the custom source information format
+   */
+  public forward(room: Room, option?: ForwardOption): Promise<boolean>
+  public forward(contact: Contact, option?: ForwardOption): Promise<boolean>
+  public forward(sendTo: Room|Contact, option?: ForwardOption): Promise<boolean> {
```

Comment:
I think again.
Initially I put the message in the message, hoping to achieve through a single function of multi-group live.
In the multi-group live operation, text, voice, pictures are more types of messages.
And only use forward () this method, you can easily achieve a variety of message types of forwarding.
So, will forward () be retained in Message or maybe? Forward () compatibility for all types of messages is ok